File: gallery-service/src/main/kotlin/com/kpi/galleryservice/domain/Project.kt
package com.kpi.galleryservice.domain

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document
import java.util.*
import javax.validation.constraints.NotBlank
import javax.validation.constraints.Size

@Document(collection = "projects")
class Project (@NotBlank @Size(max = 30) var title: String) {
    @Id
    var id: String = UUID.randomUUID().toString()
    lateinit var accountName: String
    var description = ""
}

File: neural_networks/model/project.py
from .db import db


class Project(db.Document):
    _id = db.StringField(required=True)
    title = db.StringField(required=True)
    accountName = db.StringField(required=True)
    description = db.StringField(required=True)
    _class = db.StringField()
    meta = {'collection': 'projects'}
